### DISCUSSION TITLE
0.10.7

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#6432] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.7'>0.10.7</a>.</em>

**Comment by EtherGhost** [comment]
Nice! 
Where can we find what's new?

**Reply by kekekeks** [reply]
Not really readable, but there are links to actual PRs - https://github.com/AvaloniaUI/Avalonia/compare/release/0.10.6...release/0.10.7

**Reply by EtherGhost** [reply]
Thanks!

**Comment by fforjan** [comment]
@danwalmsley @maxkatz6  following my request for 0.10.6, same here : "I really like what you did for the 0.10.5 with the list of new features & bugfixes.
Is it something you could continue to do for newer release ?"

**Comment by DrewNaylor** [comment]
Looking forward to updating the NuGet packages for a project of mine and seeing how stuff runs!

Edit Sept. 10, 2021: I've read the changelog, and the multitouch on X11 (didn't know it wasn't a thing until now, but maybe that's just Xwayland) and movable ItemsRepeater changes are probably the most exciting ones to me.

**Comment by maxkatz6** [comment]
Pushed changelog. @fforjan @kvalitetskontrollMarkaryd

**Reply by maxkatz6** [reply]
Not sure if we should have autogeneration as a CI tool or do it manually.

For this release I wrote small script https://gist.github.com/maxkatz6/1765fe9e5e0c2f09d9da5c9bc9ef4471
+ I had to assign labels on all of these pull requests
+ some minor manual edit/filtration
